kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/HelmPushMojo.java

```diff
@@ -22,24 +23,25 @@
 import static org.eclipse.jkube.kit.resource.helm.HelmServiceUtil.initHelmPushConfig;
 
 @Mojo(name = "helm-push", defaultPhase = LifecyclePhase.INSTALL, requiresDependencyResolution = ResolutionScope.COMPILE)
-public class HelmPushMojo extends HelmMojo {
+public class HelmPushMojo extends AbstractHelmMojo {
+
+  @Override
+  public void init() throws MojoFailureException {
+    super.init();
+
+    initHelmPushConfig(helm, javaProject);
+  }
 
   @Override
   public void executeInternal() throws MojoExecutionException {
-    if (skip) {
-      return;
-    }
```

Comment:
Would skip option still work after this change?

Reply:
Yes, it still works. Unit test [`HelmPushMojoTest.execute_withSkip_shouldSkipExecution()`](https://github.com/eclipse/jkube/blob/master/kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/build/HelmPushMojoTest.java#L178) checks this.
The `skip` boolean property is defined in [`AbstractJKubeMojo`](https://github.com/eclipse/jkube/blob/master/kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/AbstractJKubeMojo.java#L89). Function [`AbstractJKubeMojo.execute()`](https://github.com/eclipse/jkube/blob/master/kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/AbstractJKubeMojo.java#L156) checks the boolean before calling `executeInternal()`.